plugins/language_c7.lua

```diff
@@ -0,0 +1,80 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "cel7",
+  files = "%.c7$",
+  comment = ";",
+  patterns = {
+    { pattern = ";.-\n",              type = "comment"  },
```

Comment:
```suggestion
    { pattern = ";.*",                type = "comment"  },
```
Could you do the same for `language_fe`?